pyanaconda/modules/payloads/payloads_interface.py

```diff
@@ -30,9 +30,14 @@
 class PayloadsInterface(KickstartModuleInterface):
     """DBus interface for Payload module."""
 
-    def GetActivePayloadPath(self) -> ObjPath:
-        """Get path to the payload which is used now."""
-        return self.implementation.get_active_payload_path()
+    def GetActivePayload(self) -> ObjPath:
```

Comment:
This should be a property to be able to monitor the changes with the `PropertiesChanged` signal. Also, there should be a property that returns all created payloads. It will ensure that all created payloads are published.

Reply:
Yeah, I'm thinking if there is a reason to have this. I'm thinking if it would be better to drop the ActivePayload idea and do not recognize what payload is active.

However, there is a question then how we will solve switching between payloads. I mean if user switch from one payload to another how we will disable payloads which will not be used? 

So or so this will be better to solve in a separate PR.